src/dbnode/storage/namespace/schema.go

```diff
@@ -41,25 +41,24 @@ var (
 )
 
 type schemaDescr struct {
-	deployId string
-	md       *desc.MessageDescriptor
+	deployId     string
+	prevDeployId string
+	md           *desc.MessageDescriptor
 }
 
 func (s *schemaDescr) DeployId() string {
 	return s.deployId
 }
 
+func (s *schemaDescr) PrevDeployId() string {
+	return s.prevDeployId
+}
+
 func (s *schemaDescr) Equal(o SchemaDescr) bool {
```

Comment:
Why remove all the nil checks?

Reply:
bump

Reply:
pointer receiver should not be nil, if it is, should panic (we will fix it asap, there must be a bug in LoadSchemaRegistry). 
if o is nil, it won't pass the type assertion test in line59. so I removed those checks.